src/Sylius/Behat/Context/Ui/Admin/ManagingShipmentsContext.php

```diff
@@ -142,7 +142,7 @@ public function iShouldSeeTheShipmentOfOrderAs(string $orderNumber, string $ship
      */
     public function iShouldBeNotifiedThatTheShipmentHasBeenSuccessfullyShipped(): void
     {
-        $this->notificationChecker->checkNotification('Shipment has been successfully shipped.', NotificationType::success());
+        $this->notificationChecker->checkNotification('Shipment has been successfully updated.', NotificationType::success());
```

Comment:
Is it ok?

Reply:
Using this path instead https://github.com/Sylius/Sylius/pull/11066#discussion_r374021240 would fix the problem and will be more semantically correct